kod/object/active/holder/nomoveon/battler/monster/iceper.kod

```diff
@@ -28,9 +62,11 @@ resources:
    iceperson_rightarm = iceperR.bgf
 
    iceperson_spear = icepersp.bgf
-   iceperson_sword = icperswd.bgf
+   iceperson_sword = iceperswd.bgf
```

Comment:
You're not limited to 8.3 filenames.  Want to rename at least the new files so that they're easier to understand?